package.json

```diff
@@ -239,7 +239,7 @@
   "private": true,
   "dependencies": {
     "@department-of-veterans-affairs/formation": "^6.9.1",
-    "@department-of-veterans-affairs/formation-react": "^5.2.0",
+    "@department-of-veterans-affairs/formation-react": "5.2.6",
```

Comment:
I went ahead and just changed this to explicitly say the version based off @erikphansen's suggestion! Otherwise if I just do a `yarn` it will just grab the previous `yarn.lock` version instead of noticing the version bump. So we used to have to do:
```
rm -rf node_modules && rm yarn.lock && yarn
```
and now with this solution we can now just do:
```
yarn
```
and it will update the `yarn.lock`.

Reply:
Huh, yeah I always updated the package.json's version then did `yarn install`. I wonder if other folks were updating the version manually in the `yarn.lock` 🤷

Reply:
Actually I guess it had to be a diff to get the `integrity` property